pom.xml

```diff
@@ -1957,7 +1957,9 @@ flexible messaging model and an intuitive client API.</description>
                   <goal>report</goal>
                 </goals>
                 <configuration>
-                  <outputDirectory>target/report</outputDirectory>
+                  <excludes>
+                    <exclude>**/*.jar</exclude>
+                  </excludes>
```

Comment:
Why do you make this change?

Reply:
We just need to scan class file code coverage not jar files'

Reply:
And it may lead to some error if we scan the jar files.